src/pages/iou/IOUTransactions.js

```diff
@@ -75,7 +75,7 @@ class IOUTransactions extends Component {
         return (
             <View style={[styles.mt3]}>
                 {_.map(this.props.reportActions, (reportAction) => {
-                    if (!reportAction.originalMessage || reportAction.originalMessage.IOUReportID !== this.props.iouReportID) {
+                    if (!reportAction.originalMessage || Number(reportAction.originalMessage.IOUReportID) !== Number(this.props.iouReportID)) {
```

Comment:
NAB: I know that at some point we started using `strings` for report IDs in the App, I'm curious about why in this case we choose to convert to number to do the comparison. I'm guessing that it is because we still have numbers somewhere... is `reportAction.originalMessage.IOUReportID` a number?

I think we should leave a comment if that is the case, to avoid removing the `Number(...)` by mistake.

Reply:
Correct, one of them is still sent as number so this it why. We can add a comment just to be sure

Reply:
Sounds good. Added a comment.